add cosign verify images doc (#449)
karmada-io/website: docs/administrator/verify-artifacts.md

---

> title: Use cosign to verify signed images

Reviewer: ```suggestion
title: Verify Artifacts
```

Author: > 

ok

---

> Karmada has introduced cosign to verify the released images since version v1.6.1. The specific operation is as follows：

Reviewer: ```suggestion
Karmada has introduced cosign to verify the released images since version v1.6. The specific operation is as follows：
```

Author: > 

sorry，I may not get it. I  do't find we support v1.6.0.  In fact I was failed when I verify v1.6.0

Reviewer: Sorry, my bad. 
#3434 was merged on Jun 16, it's release-1.7.  I think we didn't sign the image for v1.6.x.

For now, people can only verify the images against the `latest` image, am I right?

Author: > Sorry, my bad. #3434 was merged on Jun 16, it's release-1.7. I think we didn't sign the image for v1.6.x.
> 
> For now, people can only verify the images against the `latest` image, am I right?

Yes,  you are right. I make an mistake here also. I will change version to `v.1.7`.

---
title: Verify Artifacts
---

## verify images

Karmada has introduced cosign to verify the released images since version v1.7. The specific operation is as follows：

### Prerequisites

You need to install the following tools:

- `cosign` ([Installation Guide](https://docs.sigstore.dev/cosign/installation/))
- `curl` (usually provided by your OS)

> - `jq` ([download jq](https://stedlan.github.io/jq/download/))

Reviewer: The link seems dead.

### Verify image signature

#### Verify image with cosign CLI

> Karmada hosts the image to `dockerhub`, and introduces the `cosign` verification tool after v1.7. For a list of published mirrors, see [karmada mirrors](https://hub.docker.com/u/karmada).

Reviewer: ```suggestion
Karmada introduced the `cosign` verification tool since release 1.7. For a list of published mirrors, see [karmada mirrors](https://hub.docker.com/u/karmada).
```
